### PR TITLE
K8SPSMDB-1013: Mongo 7.0 support

### PR DESCRIPTION
**Pull request by inelpandzic**
Add MongoDB 7.0 support

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| api-tests | passed |

File: sources/operator.1.16.0.psmdb-operator.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+            "image_path": "percona/percona-backup-mongodb:2.3.0",
+            "image_hash": "4b0a3a8dcd12619417aea16c7349cacacb41303b7131191ef65f4adff7ca1926",
+            "status": "recommended",
+            "critical": false
+          }
+        },
+        "operator": {
+          "1.16.0": {
+            "image_path": "percona/percona-server-mongodb-operator:1.16.0",
+            "image_hash": "d8a5b33db1938d42769cb5a87d34a128332a2d0302eaa6d7c860e7c4667ea3b6",
```
Comment (inelpandzic) [inline sources/operator.1.16.0.psmdb-operator.json:112]: @tplavcic what should we put here since 1.16.0 is not released?

Comment (tplavcic) [reply sources/operator.1.16.0.psmdb-operator.json:112]: I don't think we need PR for version service for mongodb 7.0 support, we can just update IMAGE_MONGOD_CHAIN in functions in the main repo.

**Review by tplavcic — CHANGES_REQUESTED** [review]

**Comment by inelpandzic** [comment]
Like @tplavcic mentioned, we don't need PR for version service for mongodb 7.0 support, we can just update IMAGE_MONGOD_CHAIN in functions in the main repo.

**Pull request closed without merging**